hollow/src/test/java/com/netflix/hollow/api/producer/VersionMinterWithCounterTest.java

```diff
@@ -0,0 +1,15 @@
+package com.netflix.hollow.api.producer;
```

Comment:
Not sure if the class VersionMinterWithCounter is being used anywhere. Is this utility requested?